undertow/src/main/java/org/wildfly/extension/undertow/ListenerResourceDefinition.java

```diff
@@ -97,6 +98,7 @@
             .build();
     protected static final SimpleAttributeDefinition ENABLED = new SimpleAttributeDefinitionBuilder(Constants.ENABLED, ModelType.BOOLEAN)
             .setAllowNull(true)
+            .setDeprecated(ModelVersion.create(3, 2))
```

Comment:
Why deprecate this? Given impl you did in this PR, enabled is well "supported" it just means different things.
It would also be wise to link write-handler of this attribute to code that can properly update the state directly to the runtime service.